plugins/modules/docker_container.py

```diff
@@ -459,8 +459,9 @@
   memory_swap:
     description:
       - "Total memory limit (memory + swap) in format C(<number>[<unit>]).
-        Number is a positive integer. Unit can be C(B) (byte), C(K) (kibibyte, 1024B),
-        C(M) (mebibyte), C(G) (gibibyte), C(T) (tebibyte), or C(P) (pebibyte)."
+        Number is a positive integer or -1 (unlimited). Unit can be C(B) (byte),
```

Comment:
How about using the value `unlimited` instead of `-1` for this?

Reply:
@felixfontein Any thoughts on allowing both `-1` and `unlimited`? `unlimited` is definitely clearer, but `-1` is used by docker itself.